### PR TITLE
Issue #560 - Introduce Global Configuration for Operator section

### LINKED ISSUE #548: Update `cloud/operator/building-custom-images.adoc`
**Summary**
This guide is no longer up to date as it references older version and information.

**Goal**
Fix the guides to be up to date

### LINKED ISSUE #560: Add an admin configuration guide for SonataFlow Operator
Related to https://github.com/apache/incubator-kie-kogito-serverless-operator/issues/312

After the mentioned feature is merged, we need a guide explaining to cluster admins how to set this new configuration and the implications of doing it.

### PR DESCRIPTION
**Pull request by ricardozanini**
Closes #560 
Closes #548

Introduces the documentation for the feature:

- https://github.com/apache/incubator-kie-kogito-serverless-operator/pull/416

**Description:**

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributions doc](https://github.com/apache/incubator-kie-kogito-docs/blob/main/CONTRIBUTING.md)
- [ ] Pull Request title is properly formatted: `Issue-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting main: `[0.9.x] Issue-XYZ Subject`
- [ ] The nav.adoc file has a link to this guide in the proper category
- [ ] The index.adoc file has a card to this guide in the proper category, with a meaningful description

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

**Comment by ricardozanini** [comment]
@kaldesai can you please take a look?

File: serverlessworkflow/modules/ROOT/pages/cloud/operator/global-configuration.adoc [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+// links
+
+This document describes how to set global configuration options for the {operator_name}.
+
+.Prerequisites
+* You have installed the operator in the target cluster. You can find more information at the xref:cloud/operator/install-serverless-operator.adoc[] guide.
+
+== Modifying configuration options
+
+After installing the operator, you can access the ConfigMap named `sonataflow-operator-controllers-config` in the namespace `{operator_installation_namespace}`.
```
Comment (domhanak) [inline serverlessworkflow/modules/ROOT/pages/cloud/operator/global-configuration.adoc:15]: `sonataflow-operator-controllers-config` 

could we make this a variable? Thinking if we ever update the name, we then just change one variable in docs.

**Review by domhanak — APPROVED** [review]
One minor comment, at your discretion.

**Comment by domhanak** [comment]
Imho this also Resolves https://github.com/apache/incubator-kie-kogito-docs/issues/548
Thanks!

**Review by kaldesai — APPROVED** [review]
The changes look good to me! Thank you!

**Review by wmedvede — APPROVED** [review]

**Comment by ricardozanini** [comment]
Merge only after:

https://github.com/apache/incubator-kie-kogito-serverless-operator/pull/416

**Pull request merged**